pkg/lib/api/response.go

```diff
@@ -22,17 +24,32 @@ import (
 type Response struct {
 	statusCode  int
 	headers     map[string][]string
+	cookies     []*http.Cookie
 	body        []byte
 	bodyControl util.JsonObject
 	Format      ResponseFormat
 }
 
+// TestCookie definition
+type TestCookie struct {
```

Comment:
Why not call this "Cookie" ? I think its in our package, so we dont need the "Test" prefix